src/components/MovieList/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import MovieItem from '../MovieItem';
+import './style.css';
+
+class MovieList extends React.Component {
+    constructor() {
+        super()
+    }
+
+    render() {
+        const { movies } = this.props;
+        const movieElements = movies.map(movie =>
+            <MovieItem key = {movie.id} movie = {movie}/>
+        )
+
+        return (
+            <div className="MovieList">
+                {movieElements}
```

Comment:
I personally would prefer to inline movieElements here :-)

Reply:
fixed, not sure if it is the best solution to put map-function in the jsx-template